lib/static/modules/actions.js

```diff
@@ -60,20 +57,28 @@ export const openDbConnection = () => {
 
             db = await mergeDatabases(dataForDbs);
         } catch (e) {
-            console.error('Error while fetching databases', e);
+            dispatch(createNotificationError('initStaticReport', e));
+        }
+
+        if (!db || isEmpty(fetchDbDetails)) {
+            return dispatch({
+                type: actionNames.INIT_STATIC_REPORT,
+                payload: {...dataFromStaticFile, db, fetchDbDetails, tree: {suites: []}, stats: [], skips: [], browsers: []}
```

Comment:
The shape of `tree` seems wrong here and leads to a runtime error in `getFailedRootSuiteIds`:

```
Uncaught (in promise) TypeError: Cannot read property 'filter' of undefined
    at getFailedRootSuiteIds (tree.js?2530:110)
    at eval (tree.js?2530:14)
    at eval (immer.esm.js?aee6:1)
    at e.i.produce (immer.esm.js?aee6:1)
    at eval (immer.esm.js?aee6:1)
    at eval (index.js?693f:30)
    at Array.reduce (<anonymous>)
    at __webpack_exports__.default (index.js?693f:25)
    at dispatch (redux.js?d77e:213)
    at eval (redux-logger.js?97fb:1)
```

To reproduce: `npm run start` in master, open http://localhost:8080/